client/src/pages/Home.js

```diff
@@ -28,12 +46,36 @@ const StyledWelcome = styled(Heading)`
   line-height: 3rem;
   margin: 2.5rem auto 0;
   text-align: center;
+
+  @media only screen and (min-width: 600px) {  
+    font-size: 2.8vw;
```

Comment:
For typography related styles, please grab them from `theme.js`. 

On a side note, let's stick with using `rem`. We currently have a root font size of `62.5%` to help with with responsiveness and scaling sizes (`1rem = 10px`).  So far we've been using `rem` for almost everything, though perhaps we should probably use `vw` and `vh` where appropriate. 

Also based on [MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units), it looks like they describe that viewport values are for things like width, height, margin, and padding (though I'm not sure how strict this is).

Reply:
I did use rem because of the same reason you mention but when I change the screen from 1440 to 1024 text size stays the same and doesn't match the design. I can write media for each different screen if that's a better option.

Reply:
Ah ok! Can you try the option of writing a media please?  I know it seems like an overkill but I think it's best to stay consistent with using `rem`.

Reply:
I did notice we also have a bunch of mediaquery sizes defined in the `constants/theme.js` and those are set in pixels
maybe these are also reusable?

Reply:
Ah didn't know about that the media queries in `theme.js`. If so let's use them

Reply:
Replaced all the value of media queries with theme.js values